mne_connectivity/tests/test_spectral.py

```diff
@@ -233,7 +277,7 @@ def test_epochs_tmin_tmax(kind):
         assert kind == 'ndarray'
         tmin = 0
         X = data
-    want_times = np.arange(n_times) / sfreq + tmin
+    # want_times = np.arange(n_times) / sfreq + tmin
```

Comment:
Would be good to restore some form of this if possible. If you want to know why it's necessary you can use blame, which git shows nicely:

https://github.com/mne-tools/mne-python/blame/main/mne/connectivity/tests/test_spectral.py#L236

You can see this was the direct result of a PR:

https://github.com/mne-tools/mne-python/pull/8861

Made in response to a bug. So yes it seems necessary to ensure... all the stuff that PR talks about.